app/helpers/two_factor_method_validator.rb

```diff
@@ -0,0 +1,30 @@
+class TwoFactorMethodValidator < ActiveModel::EachValidator
```

Comment:
Where is this class used?

Reply:
It's used in `app/forms/two_factor_options_form.rb`. I need to move it to the `/app/validators` directory. I didn't realize that directory existed until after I opened this PR. I also need to rebase.